pom.xml

```diff
@@ -268,7 +268,7 @@
             </dependency>
             <dependency>
                 <groupId>org.powermock</groupId>
-                <artifactId>powermock-api-mockito</artifactId>
+                <artifactId>powermock-api-mockito2</artifactId>
```

Comment:
if this worked before then that means none of our dependencies actually reference mockito2, or if they do they probably override the version somehow

Reply:
Local developers might have it cached, but as long as our builds don't catch any artifacts in a local Maven repo, then it's probably a signal that it's not used or overridden. Either way, I wanted to correct this without necessarily causing more havoc downstream. It's been in the common since 3.3.x, and there are a handful of sister artifacts.

Maybe PowerMock is not used with Mockito? I've used Mockito, but probably not PowerMock at the same time, and Connect uses PowerMock with EasyMock (not affected by this artifact). It is overridden in at least [one repo](https://github.com/confluentinc/kafka-connect-storage-cloud/blob/master/pom.xml#L137-L142).

@xvrl, are you suggestion we remove the definition?

Reply:
I suggest we figure out if we have any repos actually using it, if no, then let's remove it, if yes, let's make sure they don't override it